src/libs/actions/User.ts

```diff
@@ -489,7 +490,9 @@ const isChannelMuted = (reportId: string) =>
 function playSoundForMessageType(pushJSON: OnyxServerUpdate[]) {
     const reportActionsOnly = pushJSON.filter((update) => update.key?.includes('reportActions_'));
     // "reportActions_5134363522480668" -> "5134363522480668"
-    const reportIDs = reportActionsOnly.map((value) => value.key.split('_')[1]);
+    const reportIDs = reportActionsOnly
+        .map((value) => value.key.split('_')[1])
+        .filter((reportID) => reportID === Navigation.getTopmostReportId() && Visibility.isVisible() && Visibility.hasFocus());
```

Comment:
`getTopmostReportId` is expected to give only one reportID. Why should we use `filter`, we should use find in this case.

Since we use `Visibility.isVisible() && Visibility.hasFocus()`, we shouldn't even map or filter in this case, since it would be obviously failing.